release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -334,6 +334,28 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-5-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-5-deprecation-of-operatorsources"]
+==== OperatorSources and CatalogSourceConfigs block cluster upgrades
+
+OperatorSources and CatalogSourceConfigs have been deprecated for several
+{product-title} releases. Starting in {product-title} 4.4, if there are any
+custom OperatorSources or CatalogSourceConfigs objects present on the cluster,
+the `marketplace` cluster Operator set an `Upgradeable=false` condition and
+issued a *Warning* alert. This means that upgrades to {product-title} 4.5 are
+blocked if the objects are still installed.
+
+[NOTE]
+====
+Upgrades to {product-title} 4.4 z-stream releases are still permitted in this
+state.
+====
+
+Cluster administrators can see the
```

Comment:
So I get that you're identifying the user, but this just sounds a little off for me. I'd prefer to read something like "See the {product-title} 4.4 release notes for how to convert..."

Reply:
Also note the lowercase release notes